### DISCUSSION TITLE
显示配置错误

### DISCUSSION THREAD

**Discussion by sppi8** [mikumifa/biliTickerBuy#206] [提问专区]
![image](https://github.com/mikumifa/biliTickerBuy/assets/172604067/e01850a9-86e8-4e22-b9a9-07433bb2651c)
大佬救救啊

**Comment by JieKBK** [comment]
我的是出现了验证码但是点不出来

**Comment by ctrate** [comment]
同问

**Reply by sppi8** [reply]
你是不是配置的文件没有复制到抢票页面里，我也是检查才发现

**Reply by mtanjigx** [reply]
> 你是不是配置的文件没有复制到抢票页面里，我也是检查才发现

请问配置文件是哪一个
